### ISSUE TITLE
How to  run the SVM task alone

### ISSUE THREAD

**Issue by AruljothySundaramoorthy** [yassersouri/classify-text#2]
I tried to execute the main.py but it shows a single classification report, I need the SVM result and also need the mean accuracy. How can I execute to get these output?

Please reply asap

**Comment by yassersouri** [comment]
Uncomment this line: https://github.com/yassersouri/classify-text/blob/master/main.py#L98

and comment this line: https://github.com/yassersouri/classify-text/blob/master/main.py#L102

I you still have issues please post here.

**Comment by AruljothySundaramoorthy** [comment]
![github pro](https://user-images.githubusercontent.com/28552464/38595984-a19c08b2-3d6c-11e8-9292-5f96d7b2c83d.JPG)
After done the Instruction which you said earlier I am facing the error like this which I have attached below(Screenshot).

**Comment by yassersouri** [comment]
It seems like you are using a different indentation in that like of code
compared to previous ones.

**Comment by AruljothySundaramoorthy** [comment]
Yah Brother it's working, I left some space that so I found an error like that...Thanks a Lot and previously I requested you how can I find the mean accuracy of this program?

**Comment by yassersouri** [comment]
It will print it out to the console: https://github.com/yassersouri/classify-text/blob/master/main.py#L132

**Comment by AruljothySundaramoorthy** [comment]
![snip](https://user-images.githubusercontent.com/28552464/38611365-19397844-3da1-11e8-9063-662cba5b7f3b.JPG)
 I Need the mean accuracy output like this sir...!

**Comment by yassersouri** [comment]
For that you have to make some changes. I don't think I will be able to do it for you, but I can give you some pointers:
1. Instead of the `sklearn.metrics.classification_report`, use `sklearn.metrics.accuracy_score` to get the accuracy.

2. To get the mean accuracy, you have to split your dataset multiple times into different train/test splits and evaluate multiple times and report the mean and standard deviation of the accuracy. The best way to do that is to use K-Fold cross validation. http://scikit-learn.org/stable/modules/generated/sklearn.model_selection.KFold.html

**Comment by AruljothySundaramoorthy** [comment]
Thank you sir.!
